src/components/PurchaseOrder/OpenOrderConfirmationModal/OpenOrderConfirmationModal.css

```diff
@@ -0,0 +1,13 @@
+.openOrderConfirmationModalFooter {
+  display: flex;
+  justify-content: space-between;
+  flex-grow: 1;
+}
+
+.openOrderConfirmationModalFooterButton {
+  margin-bottom: 0;
+}
+
+.openOrderConfirmationModalContent {
+  text-align: center;
```

Comment:
we don't have precise mock-ups and it's ok to not implement things pixel-perfect. As far as I checked UX guidelines, all buttons were aligned to the right (so used a common wrapper `ModalFooter` from stripes-components). 
https://ux.folio.org/storybook/?selectedKind=Modal&selectedStory=ConfirmationModal&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel
https://ux.folio.org/docs/guidelines/components/modal/

Reply:
it's content, not footer with buttons

Reply:
but content should have left alignment

Reply:
right, wrong place for comment, sorry for that